posts/models.py

```diff
@@ -1,3 +1,18 @@
 from django.db import models
+from django.core.validators import MinLengthValidator
 
 # Create your models here.
+class Post(models.Model):
+    title = models.CharField(validators=[MinLengthValidator(10,"The value should be lesser than %(limit_value)s.")],max_length=100)
```

Comment:
You could use min_length parameter

Reply:
i tried min_length and its showing error  got an unexpected keyword argument 'min_length'